homeassistant/components/squeezebox/media_player.py

```diff
@@ -533,9 +594,130 @@ async def async_call_query(
         all_params = [command]
         if parameters:
             all_params.extend(parameters)
-        self._query_result = await self._player.async_query(*all_params)
-        _LOGGER.debug("call_query got result %s", self._query_result)
+
+        query_result = await self._player.async_query(*all_params)
+        _LOGGER.debug("call_query got result %s", query_result)
+        if response:
+            _res: ServiceResponse = query_result
+            return _res
+        self._query_result = query_result
         self.async_write_ha_state()
+        return None
+
+    async def async_search_service(
+        self,
+        command: str,
+        return_items: int,
+        search_string: str | None = None,
+        tags: str | None = None,
+        response: bool | None = None,
+    ) -> ServiceResponse | None:
+        """Call Squeezebox JSON/RPC method to search media library."""
+        match command:
+            case "albums":
+                _param = [
+                    "0",
+                    str(return_items),
+                    ("tags:" + tags) if tags else "tags:laay",
+                    "search:" + search_string if search_string is not None else "",
```

Comment:
It's fine to to like this instead
```suggestion
                    "search:" + search_string or "",
```

Reply:
thanks - will update.